cmdstanpy/model.py

```diff
@@ -1016,6 +1056,7 @@ def _run_cmdstan(
         """
         cmd = runset.cmds[idx]
         self._logger.info('start chain %u', idx + 1)
+        self._logger.info('threads: %s', os.environ['STAN_NUM_THREADS'])
```

Comment:
STAN_NUM_THREADS could be None?

```suggestion
        self._logger.info('threads: %s', str(os.environ.get('STAN_NUM_THREADS')))
```

Reply:
good catch - will amend logic - I don't think this needs to be reported at all, actually - will make this debug.